docs/admin/Receipt_template_editor.adoc

```diff
@@ -0,0 +1,268 @@
+Print (Receipt) Templates
```

Comment:
This whole file seems to be an accidental duplicate of receipt_template_editor.adoc (which has changes included in this pull request). Can this file be deleted (Receipt_template_editor.adoc with capital "R")?

Reply:
Yes, it can.